WorkloadManagementSystem/JobWrapper/Watchdog.py

```diff
@@ -925,10 +925,20 @@ def __setJobParamList(self, value):
 
   #############################################################################
   def getNodeInformation(self):
-    """ Attempts to retrieve all static system information, should be overridden in a subclass"""
-    methodName = 'getNodeInformation'
-    self.log.warn('Watchdog: ' + methodName + ' method should be implemented in a subclass')
-    return S_ERROR('Watchdog: ' + methodName + ' method should be implemented in a subclass')
+    """ Retrieves all static system information
+    """
+    result = {}
+    result["HostName"] = socket.gethostname()
+    result["CPU(MHz)"] = psutil.cpu_freq()[0]
+    result["Memory(kB)"] = int(psutil.virtual_memory()[1] / 1024)
+    result["LocalAccount"] = getpass.getuser()
+
+    with open("/proc/cpuinfo", "r") as cpuinfo:
+      info = cpuinfo.readlines()
+    result["ModelName"] = [x.strip().split(":")[1] for x in info if "model name" in x][0].strip()
```

Comment:
In the previous version, all whitespace was replaced, not just at the edges, but if this is only used in the list of JobParameters keeping the internal whitespace is easier to read.
```python
result["ModelName"] = info[4].split(':')[1].replace(' ', '').replace('\n', '')
```